### ISSUE TITLE
Disallow use of ghost members in non-ghost code

### ISSUE THREAD

**Issue by viper-admin** [viperproject/gobra#37]
> Created by **@arquintl** on 2020-06-19 09:45

I’ve observed that the following code snippet successfully verifies with Gobra. However, in my opinion this should only work if `h` is marked as a ghost variable.

```
package main

type Rectangle struct {
    Width, Height int
}

ghost
requires acc(r.Height)
ensures acc(r.Height)
pure func GetHeight(r *Rectangle) (res int) {
    return r.Height
}

func main() {
    r! := Rectangle{Width: 2, Height: 5}
    h := GetHeight(&r)
    assert h == GetHeight(&r) && h == 5
}
```

‌

**Comment by Felalolf** [comment]
Here is another faulty example from Peter.


[Exercise5.txt](https://github.com/viperproject/gobra/files/5306429/Exercise5.txt)

**Comment by Felalolf** [comment]
Here is another example from Peter.
[anotherExample.txt](https://github.com/viperproject/gobra/files/5306458/anotherExample.txt)

**Comment by ArquintL** [comment]
This issue might have been fixed by #239
